### PR TITLE
(#102) Add support for UUID types

### PR DESCRIPTION
**Pull request by baudoliver7**
For #102 

- Add a rule in `DefaultMappings` class
- Introduce an IT test on `PostgreSQL`
- Update documentation

### CODE REVIEW

**Comment by baudoliver7** [comment]
@yegor256 Please review / check / merge my PR.
There is one failed check (ubuntu test) about xcop not available in the path.

File: src/main/java/com/jcabi/jdbc/JdbcSession.java [hunk 1]
```diff
@@ -240,7 +241,7 @@ public JdbcSession autocommit(final boolean autocommit) {
      *
      * <p>The following types are supported: {@link Boolean},
      * {@link java.sql.Date},
-     * {@link Utc}, {@link Long}, {@link Float}, byte[], {@link Integer}.
+     * {@link Utc}, {@link Long}, {@link Float}, byte[], {@link Integer}, , {@link UUID}.
```
Comment (HuntMark) [inline src/main/java/com/jcabi/jdbc/JdbcSession.java:244]: It seems there is little a typo, isn't it? Double comma.

Comment (baudoliver7) [reply src/main/java/com/jcabi/jdbc/JdbcSession.java]: @HuntMark You are right ! Well seen. :)

**Comment by baudoliver7** [comment]
@HuntMark Please check my latest changes.

**Comment by HuntMark** [comment]
Good work :)

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) everybody who has role `REV` is banned at [#103](https://github.com/jcabi/jcabi-jdbc/issues/103); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by baudoliver7** [comment]
@HuntMark Thx :)
@yegor256 Could you check / merge please ?

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:jcabi/jcabi-jdbc#103` is not assigned, can't get performer

**Comment by baudoliver7** [comment]
@yegor256 Could you release a new version of `jcabi-jdbc` please? I need the new feature in my project :)

**Comment by andreoss** [comment]
@yegor256 @baudoliver7 I think it was merged prematurely.
https://github.com/jcabi/jcabi-jdbc/runs/2393600045

Note, that ITs run only on Linux, as other machine types does not provide Docker

**Comment by baudoliver7** [comment]
@andreoss You are right ! Thx you for fixing it in #104 :)

**Comment by yegor256** [comment]
@andreoss let's release?

**Comment by andreoss** [comment]
@rultor release, tag is `0.17.6`

**Comment by andreoss** [comment]
@baudoliver7 @yegor256 
[0.17.6](https://search.maven.org/artifact/com.jcabi/jcabi-jdbc/0.17.6/jar) released. 

Though as you can see above site publication failed (see #105).

**Comment by baudoliver7** [comment]
@andreoss Thx you very much for that release 👍

**Pull request merged**